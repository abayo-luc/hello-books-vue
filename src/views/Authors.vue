<template>
  <div class="about">
    <h1>This is the Author's page</h1>
  </div>
</template>
