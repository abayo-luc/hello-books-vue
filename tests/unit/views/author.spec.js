import {
  shallowMount
} from '@vue/test-utils';
import Authors from '../../../src/views/Authors.vue';

describe('Authors.vue', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(Authors, {
      mocks: {
        $route: {
          meta: 'main-layout'
        }
      }
    });
    expect(wrapper).toMatchSnapshot();
  });
});
