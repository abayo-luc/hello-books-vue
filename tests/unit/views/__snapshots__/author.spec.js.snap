// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authors.vue should match the snapshot 1`] = `
<div class="about">
  <h1>This is the Author's page</h1>
</div>
`;
